.editorconfig

```diff
@@ -0,0 +1,13 @@
+# https://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+
+[src/]
+indent_size = 4
```

Comment:
i have no idea why its 4 btw. i'm a 2-space man, myself.   but whatever. we'll change that another day.